alf/trainers/policy_trainer.py

```diff
@@ -690,13 +693,18 @@ def play(root_dir,
         including_optimizer=False,
         including_replay_buffer=False)
 
+    batch_size = env.batch_size
     recorder = None
     if record_file is not None:
+        assert batch_size == 1, 'video recording is not supported for parallel play'
         recorder = VideoRecorder(
             env, append_blank_frames=append_blank_frames, path=record_file)
     elif render:
-        # pybullet_envs need to render() before reset() to enable mode='human'
-        env.render(mode='human')
+        if batch_size > 1:
```

Comment:
Should instead assert batch_size==1? In this case, there is no need to create multiple envs.

Reply:
I guess we can still allow rendering if the user still want.